docs/source/index.rst

```diff
@@ -2,8 +2,11 @@ jupyter-repo2docker
 ===================
 
 **jupyter-repo2docker** is a tool to build, run, and push Docker
-images from source code repositories. See the list below for various
-ways in which you can use ``repo2docker``.
+images from source code repositories. repo2docker fetches a repo
+(e.g., from GitHub or other locations) and builds a container image based
+on the specifications found in the repo.
+Optionally, it can launch a local
+Jupyter Notebook which you can use to explore it.
```

Comment:
It can be used to explore repositories locally by building and executing the image constructed from a repository. This let's you explore the contents of a repository locally.